db/db.go

```diff
@@ -95,6 +96,15 @@ func (t *TinkDB) CheckRequiredMigrations() (int, error) {
 	return len(migrations) - len(records), nil
 }
 
+// PurgeEvents periodically checks the events table and
+// purges the events that have passed the defined EVENTS_TTL.
```

Comment:
I thought we were going to do this in the db?

Reply:
Yeah, I always forget you use the database a lot. I want to have the TTL easy to swap and configure. I imagine the TTL to be as simple as `--events-ttl 1h`, we can create a trigger with a fixed name and validate if what it does is still compliant with the TTL we set via flag @gauravgahlot ?

It looks simple to me to just start a goroutine for that, but if 🔝 can be done I think we can demand the database to do the cleaning.

I don't want to have that trigger as part of the migration because it will make it hard to configure or change

Reply:
IMHO I would prefer having something like `tink events prune`. I may accept either of the two flags - `--all` and `--ttl`. 
So, we could either do `tink events prune --all` or `tink events prune --ttl 60m`.

Reply:
it is not a client feature, it is a server one

Reply:
In the end we use a separate goroutine and I think we are holding this work for too much, this is the easiest implementation and the repo is always open for new contributions and evolution 👍

Reply:
yep that sound fine, just wanted to make sure I understood where things stood.